<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BF4Widget</class>
 <widget class="QWidget" name="BF4Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_si_level">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_si_restartRound">
       <property name="toolTip">
        <string>Restart round</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../frostbite/Frostbite.qrc">
         <normaloff>:/frostbite/icons/level-restart.png</normaloff>:/frostbite/icons/level-restart.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_si_runNextRound">
       <property name="toolTip">
        <string>Run next round</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../frostbite/Frostbite.qrc">
         <normaloff>:/frostbite/icons/level-next.png</normaloff>:/frostbite/icons/level-next.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_si_round">
       <property name="text">
        <string>Round:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_si_players">
       <property name="text">
        <string>Players:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_si_version">
       <property name="text">
        <string>Version:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_si_upTime">
       <property name="text">
        <string>Uptime:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="windowTitle">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../frostbite/Frostbite.qrc"/>
 </resources>
 <connections/>
</ui>
